pkg/controller/installation/core_controller_test.go

```diff
@@ -1274,4 +1274,375 @@ var _ = Describe("Testing core-controller installation", func() {
 			})
 		})
 	})
+	Context("Installation CR's Status conditions tests", func() {
```

Comment:
You can move this describe inside of the context called "Reconcile tests". Then you can delete the BeforeEach and AfterEach from this one. That way we do not need all this code duplication.